projects/plugins/jetpack/tools/webpack.config.extensions.js

```diff
@@ -71,7 +76,7 @@ const viewBlocksScripts = presetBetaBlocks.reduce( ( viewBlocks, block ) => {
 // Combines all the different production blocks into one editor.js script
 const editorScript = [
 	editorSetup,
-	...blockScripts(
+	...blockEditorScripts(
```

Comment:
This is also a nitpick, but should we be renaming `presetProductionBlocks` to something like `presetProductionExtensions` now that it includes plugins?